app/code/Magento/CustomerImportExport/Model/Import/Customer.php

```diff
@@ -249,6 +255,7 @@ public function __construct(
         $this->_initStores(true)->_initAttributes();
 
         $this->_customerModel = $customerFactory->create();
+        $this->_indexerRegistry = $indexerRegistry;
```

Comment:
Please, init the `indexerRegistry` using object manager in case if no value is passed to the corresponding constructor argument. More details [here](https://devdocs.magento.com/guides/v2.3/contributor-guide/backward-compatible-development/) ("Adding a constructor parameter" section)

Reply:
Just added this in the new commit. Thank you very much for the review :)